README.org

```diff
@@ -748,6 +750,33 @@ The above code makes the backend available to select.  If you want it to be the
 
 #+html: </details>
 #+html: <details><summary>
+
+**** Sambanova (Deepseek)
+#+html: </summary>
+Sambanova offers various LLMs through their Samba Nova Cloud offering, with Deepseek-R1 being one of them. The token speed for Deepseek R1 via Sambanova is about 6 times faster than when accessed through deepseek.com 
+
+Register a backend with
+#+begin_src emacs-lisp
+(gptel-make-openai "Sambanova"        ;Any name you want
+  :host "api.sambanova.ai"
+  :endpoint "/v1/chat/completions"
+  :stream t                          ;for streaming responses
+  :key "your-api-key"               ;can be a function that returns the key
+  :models '(DeepSeek-R1))
+#+end_src
+
+You can pick this backend from the menu when using gptel (see [[#usage][Usage]]).
+
+***** (Optional) Set as the default gptel backend
+The code aboves makes the backend available for selection.  If you want it to be the default backend for gptel, you can set this as the value of =gptel-backend=.  Add these two lines to your configuration: 
+#+begin_src emacs-lisp
+;; OPTIONAL configuration
+  (setq gptel-model 'DeepSeek-R1)
+  (setq gptel-backend (cdr (assoc "Sambanova" gptel--known-backends)))
```

Comment:
`gptel--known-backends` is an internal variable.  Use `(gptel-get-backend "Sambanova")` instead.